jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -28,8 +31,11 @@ class Markers(PluginTemplateMixin, ViewerSelectMixin, TableMixin):
     * :meth:`~jdaviz.core.template_mixin.TableMixin.clear_table`
     * :meth:`~jdaviz.core.template_mixin.TableMixin.export_table`
     """
+    distance_display = traitlets.Unicode("N/A").tag(sync=True)
     template_file = __file__, "markers.vue"
-    uses_active_status = Bool(True).tag(sync=True)
+    uses_active_status = traitlets.Bool(True).tag(sync=True)
+
+    distances_table = traitlets.Instance(Table).tag(sync=True, **widgets.widget_serialization)

```

Comment:
looking forward to when we might have multiple line segments, or even closed polygons, would we want to build that same functionality on top of this same table?  Would that motivate a more general name?